### PR TITLE
DO NOT MERGE: Adding kneaddata pipeline

### PR DESCRIPTION
**Pull request by josenavas**

### CODE REVIEW

**Comment by josenavas** [comment]
Retest this please

**Comment by josenavas** [comment]
retest this please

**Comment by jdereus** [comment]
retest this please

**Comment by jdereus** [comment]
retest this please

File: .travis.yml [hunk 1]
```diff
@@ -13,14 +13,17 @@ install:
...
-    'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
+    'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1' matplotlib
   - source activate qiita_env
   - pip install sphinx sphinx-bootstrap-theme coveralls ipython[all]==2.4.1
   - pip install https://github.com/biocore/qiita/archive/master.zip --process-dependency-links
   # installing other deps
   - conda install --yes -c jjhelmus glpk
   - conda install --yes -c bioconda diamond bowtie2 trimmomatic=0.36 fastqc=0.11.5
   #
+  - which trimmomatic
```
Comment (jdereus) [inline .travis.yml]: call of trimmomatic spelling error causing build to fail.  can you fix?

File: .travis.yml [hunk 2]
```diff
@@ -13,14 +13,17 @@ install:
...
+  - echo $PATH
+  - triommomatic
```
Comment (jdereus) [inline .travis.yml:26]: incorrect comment location for previous.  typo here for trimmomatic call.

**Comment by antgonza** [comment]
closing in favor of https://github.com/qiita-spots/qp-shotgun/pull/19

**Pull request closed without merging**